src/styles.scss

```diff
@@ -1 +1,70 @@
 /* You can add global styles to this file, and also import other style files */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: var(--surface-ground);
+}
+
+:root {
+  --text-color: #1e293b;
+  --text-color-secondary: #64748b;
+  --primary: #3b82f6;
+  --cyan-500: #06b6d4;
+  --cyan-100: #c3edf5;
+  --purple-500: #a855f7;
+  --purple-100: #ead6fd;
+  --success: #22c55e;
+  --alert: #f59e0b;
+  --table-header: #f8fafc;
+  --sufrace-border: #dfe7ef;
+  --surface-card: #ffffff;
+  --surface-ground: #eff3f8;
+}
+
+.default-card {
+  background-color: var(--surface-card);
+  padding: 10px 28px;
+  border: 1px solid var(--sufrace-border);
+  border-radius: 10px;
+  align-items: center;
+}
+
+// ************ Helpes Classes ************
+
+.flexbox-space-between {
+  display: flex;
+  justify-content: space-between;
+}
+
+.time-bold {
+  font-family: "Chivo Mono";
+  font-weight: 700;
+}
+
+.text-semi-bold {
+  font-weight: 600;
+}
+
+.p-dropdown {
```

Comment:
Later the prime-ng modifications can go into a separated scss file and imported here.

Reply:
I know the concept but later on we should talk about that how would you like to sort scss files